src/pages/Home/components/Content/FilterBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Genre } from '../../../../shared/interfaces';
+import './FilterBar.css';
+
+type ButtonName = 'All' | Genre;
+export const FilterBar = () => (
+  <div className='filterBar'>
+    <nav>
+      <GenreButtons />
+    </nav>
+    <div className='sortContainer'>
+      <label>Sort By</label>
+      <select>
+        <option>Release Date</option>
+      </select>
+    </div>
+  </div>
+);
+
+function GenreButtons() {
+  const buttonNames: ButtonName[] = [
```

Comment:
Try to use enums - you will not need to duplicate type possible values into the array

Reply:
done. thx